version/version.go

```diff
@@ -33,12 +33,6 @@ func (version) Usage() string {
 }
 
 func (version) Execute(context extpoints.Context) bool {
```

Comment:
This is a good point, although the fix is nicely isolated in its own commit, which is often enough for something fairly simple.